playbooks/roles/artifacts/tasks/minishift.yml

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
different role name besides artifacts

Reply:
This is the artifacts of 'minishift'. Exist the artifacts to 'oc'. This is only to separate what the relationship of artifacts. Maybe in the future, we can download other artifacts. I had difficult to understand and debug the problems in dev_setup because this I'm trying separate most is possible. Small files and easy to maintain and debug.